Tests/BootCamp.Chapter.Tests/ProgramTests.cs

```diff
@@ -17,17 +17,27 @@ public class ProgramTests : IDisposable
         [InlineData("nonExisting.csv")]
         public void Main_When_Transactions_File_Empty_Or_Not_Found_Throws_NoTransactionsException(string input)
         {
-            Action action = () => Program.Main(new[] { input });
+            const string cmd = "time";
+
+            Action action = () => Program.Main(new[] { input, cmd, OutputFile });
 
             action.Should().Throw<NoTransactionsFoundException>();
         }
 
+        [Fact]
+        public void Main_When_Too_Few_Arguments_Throws_InvalidCommandException()
```

Comment:
Added to ensure program is invoked with the minimum number of arguments. Created the need to modify `Main_When_Transactions_File_Empty_Or_Not_Found_Throws_NoTransactionsException`.

Reply:
Yes, looks good.